boilerplate/flyte/end2end/run-tests.py

```diff
@@ -98,10 +98,15 @@ def executions_finished(executions_by_wfgroup: Dict[str, List[FlyteWorkflowExecu
     return True
 
 def sync_executions(remote: FlyteRemote, executions_by_wfgroup: Dict[str, List[FlyteWorkflowExecution]]):
-    for executions in executions_by_wfgroup.values():
-        for execution in executions:
-            print(f"About to sync execution_id={execution.id.name}")
-            remote.sync(execution)
+    try:
+        for executions in executions_by_wfgroup.values():
+            for execution in executions:
+                print(f"About to sync execution_id={execution.id.name}")
+                remote.sync(execution)
+    except:
+        print("GOT TO THE EXCEPT")
```

Comment:
remove this?

Reply:
This is auto-generated by the script. I will update the file in flyteorg/boilerplate as well. https://github.com/flyteorg/boilerplate/blob/master/boilerplate/flyte/end2end/run-tests.py#L100-L108

Reply:
oh I see! was this change in boilerplate intentional?